### DISCUSSION TITLE
External Recipes Local Backend Import Sub Folder or Folder with Spaces

### DISCUSSION THREAD

**Discussion by cdesouza71** [TandoorRecipes/recipes#1202] [Q&A]
Hello,

nice software, thank you!

Is it possible to import a directory hierarchy using the local backend or do I have to set up a new watch folder for each sub directory? 

Also when I have a watch directory with spaces such as "Hauptgericht - Pasta"  and I press "Sync Now!" I get a Server Error (500). The Recipes are however displayed in the Discovery List

**Comment by vabene1111** [comment]
the external recipes feature has sadly not seen much love in the last few month as most work has been focused on the internal ones.

i honestly dont know how the importer behaves on subdirectories, i would assume they are ignored and need seperate importers but would need to check this in the code myself. At some point i will revisti the whole external recipes system and fix all those nasty little rough edges.

**Reply by cdesouza71** [reply]
Hello,

subdirectories are ignored.
